app/views/components/table_row.scala.html

```diff
@@ -19,9 +19,16 @@
 
 @(id: String, tableRow: HtmlTableRow)(implicit messages: Messages)
 
-@changeLink(callOpt: Option[Call]) = {
+@changeLink(callOpt: Option[Call], changeLabel: Option[String]) = {
```

Comment:
Can the changeLabel be non-optional?    Currently if it's None then the "Change" text will be "aria-hidden" with nothing to replace it.

Reply:
Yes